### ISSUE TITLE
Series idxmin() TypeError?Need change type to 'float'

### ISSUE THREAD

**Issue by rICHi97** [pandas-dev/pandas#41696]
i'm using idxmin() on a series, which is like:
```
    ltc: 
    调度报障工单号
    DD182103227    5
                  ..
    DD082104265    5
    Name: 供电所, Length: 848, dtype: Int64
```
However, when using:
`    ltc.idxmin()`

i got an error:
`    TypeError: argmin() takes 1 positional argument but 2 were given`

when using:
`  ltc.astype('float').idxmin()`

i got the right answer:
    'DD092104104'

But the api doesn't say that idxmin() can only use on float data.Why?

**Comment by ZurMaD** [comment]
I tried to replicate the error but can't get it.

INPUT:
```python
import pandas as pd
import numpy as np

df=pd.DataFrame({'col':['DD182103227','DD082104265'],
                'value':[5,4]},dtype=np.int64)
df.set_index('col')
```
OUTPUT:
```python
             value
col               
DD182103227      5
DD082104265      4
```

INPUT:
```python
df.info()
```
OUTPUT:
```python
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2 entries, 0 to 1
Data columns (total 2 columns):
 #   Column  Non-Null Count  Dtype 
---  ------  --------------  ----- 
 0   col     2 non-null      object
 1   value   2 non-null      int64 
dtypes: int64(1), object(1)
memory usage: 160.0+ bytes
```

INPUT:
```python
df['value'].idxmin()
```
OUTPUT:
```
1
```

INPUT:
```python
df['value'].astype('float').idxmin()
```
OUTPUT:
```
1
```

**Comment by rICHi97** [comment]
>

**Comment by rICHi97** [comment]
@ZurMaD 
Thanks for your reply.
There is a difference between your test data and mine:
my data contains missing value, <NA>.
Tyr this code:
IN:
`test = pd.Series([1, 2, None], dtype='Int64')`
OUT:
`
0       1
1       2
2    <NA>
dtype: Int64
`
IN:
`test.idxmin()`
OUT:
`TypeError: argmin() takes 1 positional argument but 2 were given`

Maybe integer dtypes and missing data causes this problem. But idxmin provides a _skipna_ param and default is True, I don't know why it can't handle integer data.

**Comment by ZurMaD** [comment]
Duplicate of #33719 

and with a fork in https://github.com/pandas-dev/pandas/pull/27801

**Comment by mzeitlin11** [comment]
This was fixed by #37924 with tests. (Fix is only in master, will be in 1.3)
